### ISSUE TITLE
Infinite refresh loop on landing page

### ISSUE THREAD

**Issue by TheKrush** [cynicaloptimist/improved-initiative#298]
If I forcibly changed the url to give it an encounter and then click End Tutorial the next time I load the landing page everything is ok.

So I guess something in the cookies is stopping the infinite refresh (it is reproducible by just clearing the cookies and data for the site).

**Comment by TheKrush** [comment]
I just confirmed it stops refreshing immediately as soon as 'End Tutorial' is clicked. It seems that this action sets something in local storage which prevents it from reloading forever.

Deleting the local storage is causing this to show up.

Worth noting I'm running the site without a database and set account level to epic initiative. Maybe it's something related to those as well.

**Comment by cynicaloptimist** [comment]
I think I know what's happening here.

To mitigate #207, I added a check when you visit the landing page that attempts to redirect you to the Canonical URL if this is your first visit. I neglected to write fallback behavior for if there is no Canonical URL (aka the app setting BASE_URL). The quick fix is to set that app setting, a more correct fix is to handle an empty url by not [redirecting at page load](https://github.com/cynicaloptimist/improved-initiative/blob/master/client/LauncherViewModel.ts#L16)
